### ISSUE TITLE
LDAP Filter 

### ISSUE THREAD

**Issue by julien3400** [jellyfin/jellyfin-plugin-ldapauth#144]
Hello can someone help me I am on openLdap
the server and reachable but I do not understand why my filters do not work it puts me Found 0 user(s), 0 admin(s) with this filter the (memberOf=CN=jellyfin_users,ou=groups,,DC=home, DC=local)
and admin filter
(memberOf=cn=jellyfin_admins,ou=groups,dc=home,dc=local)

Thanks for your help

**Comment by Roncarlos** [comment]
![image](https://github.com/jellyfin/jellyfin-plugin-ldapauth/assets/9269672/6cbf375c-e5bb-4d4d-9ab4-dda7c128579d)
![image](https://github.com/jellyfin/jellyfin-plugin-ldapauth/assets/9269672/f2804964-8541-403a-bb44-9c1956239ca8)
![image](https://github.com/jellyfin/jellyfin-plugin-ldapauth/assets/9269672/1b9b376d-b1f4-413a-a7c5-5f46d36f100b)

Is this normal behaviour? Looks like it's related to the problem in this issue.

**Comment by Roncarlos** [comment]
I'm also using Authentik. (It's not working on either version 15 or 16 of the plugin) so it's maybe related to the implementation of LDAP by Authentik?

**Comment by DarkVirtue** [comment]
@Roncarlos Yes, that seems to be the case. authentik version [2023.6.1](https://goauthentik.io/docs/releases/2023.6#fixed-in-202361) just released and included a bunch of fixes for LDAP. Jellyfin LDAP  is getting all the correct results after I updated authentik.
